src/command.lisp

```diff
@@ -1172,6 +1172,74 @@ _~~A() {
           (format stream "~A~%" line)))
       (format stream "~%"))))
 
+(defmethod print-documentation ((kind (eql :org)) (top-level command) stream &key (wrap-at 80))
+  "Prints the documentation for the given TOP-LEVEL command in Org Mode format"
+  (with-command-tree (node top-level)
+    ;; Initialize command, so that options get propagated
+    (initialize-command node)
+
+    ;; Command name
+    (format stream "#+TITLE: ~A~2%" (command-full-name node))
+
+    ;; Print description
+    (cond
+      ;; Print long description
+      ((command-long-description node)
+       (let ((lines (split-sequence #\Newline
+                                    (bobbin:wrap (command-long-description node) wrap-at))))
+         (dolist (line  lines)
+           (format stream "~A~%" line))
+         (format stream "~%")))
+      ;; Print short description only
+      (t (format stream "=~A= -- ~A~2%" (command-full-name node) (command-description node))))
+
+    ;; Usage info
+    (format stream "* Usage~2%")
+    (format stream "#+begin_src shell~%~A~%#+end_src~2%" (command-usage-string node))
+
+    ;; Options
+    (when (command-options node)
+      (format stream "* Options~2%")
+      (format stream "=~A= accepts the following options:~2%" (command-full-name node))
+      (format stream "#+begin_src shell~%")
+      (print-options-usage node stream)
+      (format stream "#+end_src~2%"))
+
+    ;; Sub-commands
+    (when (command-sub-commands node)
+      (format stream "* Sub Commands~2%")
+      (format stream "=~A= provides the following sub commands:~2%" (command-full-name node))
+      (format stream "#+begin_src shell~%")
+      (print-sub-commands-info node stream)
+      (format stream "#+end_src~2%"))
+
+    ;; Examples
+    (when (command-examples node)
+      (format stream "* Examples~2%")
+      (dolist (example (command-examples node))
+        (let* ((description (car example))
+               (code (cdr example))
+               (lines (split-sequence #\Newline (bobbin:wrap description wrap-at))))
+          (dolist (line lines)
+            (format stream "~A~%" line))
+          (format stream "~%")
+          (format stream "#+begin_src shell~%~A~%#+end_src~2%" code))))
+
+    ;; Authors
+    (when (command-authors node)
+      (format stream "* Authors~2%")
+      (dolist (author (command-authors node))
+        (format stream "* ~A~%" author))
```

Comment:
Should probably be `- ~A~%` to make it a list, otherwise it will become another header in org-mode.